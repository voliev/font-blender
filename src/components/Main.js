import React from "react";
import FontsPane from "./fontsPane/FontsPane";
import ShowcasePane from "./showcasePane/ShowcasePane";
import SettingsPane from "./settingsPane/SettingsPane";
import "./styles/Main.css";

const Main = () => (
  <main className="Main">
    <FontsPane />
    <ShowcasePane />
    <SettingsPane />
  </main>
);

export default Main;
