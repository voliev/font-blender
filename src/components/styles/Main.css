.Main {
  display: flex;
  flex-direction: row;
  height: calc(100% - 2.074rem * 1.2 * 1.5);
  width: 100%;
}
