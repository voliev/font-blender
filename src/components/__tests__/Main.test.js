import React from "react";
import { shallow } from "enzyme";
import Main from "../Main";

it("should render without crashing", () => {
  const wrapper = shallow(<Main />);
  expect(wrapper).toMatchSnapshot();
});

it("should render children components", () => {
  const wrapper = shallow(<Main />);
  expect(wrapper.children().length).toBe(3);
});
