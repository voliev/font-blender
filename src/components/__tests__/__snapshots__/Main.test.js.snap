// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<main
  className="Main"
>
  <FontsPane />
  <Connect(ShowcasePane) />
  <SettingsPane />
</main>
`;
